62.CAD/vertex_shader.hlsl

```diff
@@ -181,6 +232,16 @@ ClipProjectionData getClipProjectionData(in MainObject mainObj)
     }
 }
 
+inline float2 quadraticDerivative(float t, float2 A, float2 B, float2 C)
+{
+    return 2.0f*A*t + B;
+}
+
+inline float2 quadraticSecDerivative(float t, float2 A, float2 B, float2 C)
+{
+    return 2.0f*A;
+}
```

Comment:
are you even using these? if yes move them to quadratic bezier functions in beziers.hlsl

Reply:
done